ci/images/e2e-test-runner/run-e2e-test.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+set -o errexit
```

Comment:
That script might be bit long. You may want to consider to break it up into a few functions to help with readability. You may keep it as it is if you prefer, as to me it's in the gray area.

Reply:
Good idea. I will improve it. This is not easy to verify it for now. If I couldn't get it done before this PR is merged. I will open a new PR for the improvement.